python/lsst/meas/astrom/astrometry.py

```diff
@@ -118,8 +119,8 @@ class AstrometryTask(RefMatchTask):
     ConfigClass = AstrometryConfig
     _DefaultName = "astrometricSolver"
 
-    def __init__(self, refObjLoader, schema=None, **kwargs):
-        RefMatchTask.__init__(self, refObjLoader, **kwargs)
+    def __init__(self, refObjLoader=None, schema=None, **kwargs):
+        RefMatchTask.__init__(self, refObjLoader=refObjLoader, **kwargs)
```

Comment:
It'd be nice to actually deprecate this argument, but certainly out of scope for this ticket.

Reply:
I'm not sure we actually want to deprecate it: I could use uses of it in e.g. notebooks.